bitmind/download_data.py

```diff
@@ -0,0 +1,93 @@
+from datasets import load_dataset
+import argparse
+import time
+import sys
+import os
+import subprocess
+import glob
+
+from bitmind.constants import DATASET_META, HUGGINGFACE_CACHE_DIR
```

Comment:
Should just be `from constants import ...` not `bitmind.constants` after `download_data.py` has been moved

Reply:
^ Ignore this. Was not an issue after I reran pip requirement installs